### DISCUSSION TITLE
Mage the Ascension, Sphere Specializations

### DISCUSSION THREAD

**Discussion by AgentDelirium** [JohanFalt/Foundry_WoD20#821] [Q&A] [answered]
Hello. Loving the system for foundry. Works fantastic, but in the recent update, we lost the ability to give characters sphere specializations for people with 4 or more dots. Is that now a toggle-able function, or was it accidentally removed?

**Comment by JohanFalt** [comment]
This is a bug I’ll fix it.

**Comment by JohanFalt** [comment, marked as answer]
Fix in 3.3.1
